tests/UnitTests/Foundation/WindowsOnly/BundleInternalTests.mm

```diff
@@ -30,20 +30,13 @@ virtual void SetUp() {
         NSString* _subDirectory = @"en.lproj";
         NSString* localizationResourceName = @"Localizable.strings";
         // Create a unique test directory
-        NSString* tempDir = [[@"./tmp_TestFoundation" stringByAppendingString:[NSUUID UUID].UUIDString] stringByAppendingString:@"/"];
-
+        NSString* tempDir = [[@"./tmp_TestFoundation" stringByAppendingPathComponent:[NSUUID UUID].UUIDString] stringByAppendingPathComponent:@"/"];
+        NSString* bundlePath = [tempDir stringByAppendingPathComponent:_bundleName];
```

Comment:
funny question: why not just expose bundlePath in the fixture and use that instead of re-combining `_playground` and `_bundleName` later?

Reply:
Not funny at all. That seems like a good idea to avoid extra string work. 👍